.github/workflows/ci.yml

```
@@ -263,7 +263,7 @@ jobs:
 
       - name: Get test matrix
         run: |
-          if .github/scripts/has_riscv_changes.sh; then
+          if "${{ github.ref_name }}" == "main" && .github/scripts/has_riscv_changes.sh; then
```

This is fed to the shell, and its if syntax tries to run github.ref_name as command:
```
Run if "188/merge" == "main" && .github/scripts/has_riscv_changes.sh; then
/__w/_temp/b2e3ad64-7590-4c9c-bcda-01226f35575f.sh: 1: 188/merge: not found`
```

I think the correct syntax should be:
```suggestion
          if [[ "${{ github.ref_name }}" == "main" ]] && .github/scripts/has_riscv_changes.sh; then
```

---

You'd think I'd learn after writing so many hack-scripts..